### DISCUSSION TITLE
Erro ao acessar Área do Cliente - Whoops\Exception\ErrorException (E_WARNING) Undefined variable $configuration

### DISCUSSION THREAD

**Discussion by mtecinformatica** [RamonSilva20/mapos#2062] [General]
Ótimo dia pessoal! Como vocês estão?
Por gentileza alguém pode me ajudar com um problema ao acessar a área do cliente? Estou com o MapOS configurado na hospedagem da Hostigator e ao tentar acesso a área do cliente ocorre o erro: 

Whoops \ Exception \ ErrorException (E_WARNING)
Undefined variable $configuration

Segue print da tela com o erro pra ser mais específico.
![11-02-2023 10_06_26](https://user-images.githubusercontent.com/125134172/218259999-3a01a550-fbda-4bce-9821-555d67936303.jpg)

**Comment by willph** [comment]
Olá. Tudo bom? 

Qual a versão do php está usando?

Aconselho php 7.4
Php8 ou superior não funciona por enquanto

**Reply by mtecinformatica** [reply]
Opa, ótimo dia!

Me parece que está no PHP 7.4.30

![11-02-2023 10_33_05-OS-08 02 2023](https://user-images.githubusercontent.com/125134172/218260830-a4414643-b9ee-4812-b8e6-bfc1520336c7.jpg)

**Comment by willph** [comment]
Seu servidor é multiphp?

Se for, tem que olhar qual versão do php está usando no domínio ou subdomínio que está o MapOs. 

Verificar se nele está versão 7.4

**Comment by johnrickaraujo** [comment]
Olá @mtecinformatica Tive o mesmo problema.
Eu uso a versão 4.30.2 No subdomínio admin. (Tinha feito umas alterações, com um colega).
Essa semana fui rodar a ultima versão para testar, deu esse erro...
Quebrei a cabeça algumas horas, até ir nessa opção e trocar a versão PHP 8 para 7...
Resolveu.

Painel Cpanel do Domínio Principal:

![PHP-Hostgator](https://user-images.githubusercontent.com/85075741/221992381-4f82febe-e9b9-4a9f-90fe-890f32396f81.jpg)

Depois de alterado fucnionou:

![PHP-versoes](https://user-images.githubusercontent.com/85075741/221992522-6a7402c8-f967-4dcc-8cd7-e4d4992768bf.jpg)

Se fizer e não der certo, me manda um e-mail, te ajudo...
john@mdetecnologia.com
Sei menos que o básico, mas consigo te ajudar a ficar online...

**Reply by mtecinformatica** [reply]
Opa, ótimo dia! Cara você me ajudou muito... o suporte da hospedagem dizia e batia o pé que estava na versão 7. Segui os prints que você mandou e alterei lá e deu certo! Muito obrigado pela ajuda.

**Reply by johnrickaraujo** [reply]
Top Manolo!!! Boa sorte ai no projeto! Abraço...
